packages/main/src/SegmentedButtonItem.ts

```diff
@@ -30,63 +34,79 @@ import { SEGMENTEDBUTTONITEM_ARIA_DESCRIPTION } from "./generated/i18n/i18n-defa
+	renderer: litRender,
 	template: SegmentedButtonItemTemplate,
+	styles: segmentedButtonItemCss,
 	dependencies: [Icon],
 })
-class SegmentedButtonItem extends ToggleButton implements ISegmentedButtonItem {
+class SegmentedButtonItem extends UI5Element implements IButton, ISegmentedButtonItem {
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default "Default"
+	 * Defines whether the component is disabled.
+	 * A disabled component can't be pressed or
+	 * focused, and it is not in the tab chain.
+	 * @default false
 	 * @public
 	 */
-	@property({ type: ButtonDesign, defaultValue: ButtonDesign.Default })
-	declare design: `${ButtonDesign}`;
+	@property({ type: Boolean })
+	disabled!: boolean;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
+	 * Determines whether the component is displayed as selected.
 	 * @default false
 	 * @public
 	 */
 	@property({ type: Boolean })
-	declare iconEnd: boolean;
+	selected!: boolean;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default false
+	 * Defines the tooltip of the component.
+	 *
+	 * **Note:** A tooltip attribute should be provided for icon-only buttons, in order to represent their exact meaning/function.
+	 * @default ""
 	 * @public
+	 * @since 1.2.0
 	 */
-	@property({ type: Boolean })
-	declare submits: boolean;
+	@property()
+	tooltip!: string;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default {}
+	 * Defines the icon, displayed as graphical element within the component.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * Example:
+	 * See all the available icons within the [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html).
+	 * @default ""
 	 * @public
 	 */
-    @property({ type: Object })
-    declare accessibilityAttributes: AccessibilityAttributes;
+	@property()
+	icon!: string;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default "Button"
-	 * @public
+	 * Indicates if the element if focusable
+	 * @private
 	 */
-	@property({ type: ButtonType, defaultValue: ButtonType.Button })
-	declare type: `${ButtonType}`;
+	@property({ type: Boolean })
+	nonInteractive!: boolean;
 
 	/**
-	 * **Note:** The property is inherited and not supported. If set, it won't take any effect.
-	 * @default "Button"
-	 * @public
+	 * Defines the tabIndex of the component.
+	 * @private
 	 */
-	@property({ type: ButtonAccessibleRole, defaultValue: ButtonAccessibleRole.Button })
-	declare accessibleRole: `${ButtonAccessibleRole}`;
+	@property({ defaultValue: "0", noAttribute: true })
+	forcedTabIndex!: string;
+
+	/**
+	 * Indicates if the elements is on focus
+	 * @private
+	 */
+	@property({ type: Boolean })
+	focused!: boolean;
```

Comment:
Don't think that this property is really needed after the following PR: https://github.com/SAP/ui5-webcomponents/pull/8414/files#diff-6b143fe8e8ac23509b7a00b63e8d54a362076da658753aefcc9d01bb290c1e9f
This could make the `_onfocusin` and `_onfocusout` handlers redundant.